src/formatDate.js

```diff
@@ -8,7 +8,46 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+
+  const dateInOldFormatArr = date.split(oldSeparator);
+  const dateInNewFormatArr = [];
+
+  const month = dateInOldFormatArr[fromFormat.indexOf('MM')];
+  const day = dateInOldFormatArr[fromFormat.indexOf('DD')];
+  let year;
+
+  if (date.length === 10) {
+    year = dateInOldFormatArr[fromFormat.indexOf('YYYY')];
+    year = `${year[2]}${year[3]}`;
```

Comment:
When converting from 'YYYY' to 'YY', you are only taking the last two digits of the year. However, you should also consider the case when converting to 'YYYY'. Use the `fromFormat` and `toFormat` arrays to determine the correct conversion.